### ISSUE TITLE
working with async methods.

### ISSUE THREAD

**Issue by Misterhex** [Pablissimo/nrconfig#10]
is there a command flag to work with async methods instrumentation?

**Comment by Pablissimo** [comment]
Nope, when this was first created I don't think async/await had made it to C# yet. Will cheerily accept a pull request but I've not got access to New Relic any more so would struggle to develop and test myself

**Comment by Misterhex** [comment]
@Pablissimo thanks for getting back
